ouroboros-network/test/Test/Subscription.hs

```diff
@@ -29,7 +29,7 @@ import           Data.Int
 import qualified Data.IP as IP
 import           Data.List as L
 import qualified Data.Map as M
-import           Data.Time.Clock
+import           Data.Time.Clock ()
```

Comment:
I'm curious, which instances are needed here?

Reply:
Oops, that was temporary. I was checking nothing was used and then forgot.